rector.php

```diff
@@ -18,5 +18,6 @@
         ShopwareSetList::SHOPWARE_6_6_4,
         ShopwareSetList::SHOPWARE_6_6_10,
         ShopwareSetList::SHOPWARE_6_7_0,
+        ShopwareSetList::SHOPWARE_6_7_8,
```

Comment:
This looks wrong

Reply:
Ah yes, you are right 🙈 looking forward to my holiday next week 😊 

```suggestion
        ShopwareSetList::SHOPWARE_6_8_0,
```